pkgs/development/tools/kustomize/default.nix

```diff
@@ -3,9 +3,9 @@
 
 buildGoPackage rec {
   name = "kustomize-${version}";
-  version = "1.0.9";
-  # rev is the 1.0.8 commit, mainly for kustomize version command output
-  rev = "ec86b30d2b01a8fa62e645f024f26bfea5dcd30d";
+  version = "1.0.10";
+  # rev is the 1.0.10 commit, mainly for kustomize version command output
+  rev = "383b3e798b7042f8b7431f93e440fb85631890a3";
```

Comment:
Not I understand why we don't use the `v1.0.10` release tag

Reply:
@c0bw3b it's to populate the `gitCommit` field of the `version` output (as the published releases)